systemd/systemd.go

```diff
@@ -733,28 +733,7 @@ func MountUnitPath(baseDir string) string {
 
 var squashfsFsType = squashfs.FsType
```

Comment:
I notice FsType returns an error that is always nil, maybe given we are touching this we can simplify that away, it means some new helpers here don't need to return an error either

Reply:
Indeed, thanks for spotting, done.